src/modules/bh_conference.erl

```diff
@@ -21,15 +21,30 @@
 
 -include("blackhole.hrl").
 
+-define(CONFERENCE_EVENTS
+       ,[<<"*">> | ecallmgr_conference_event_publisher:events()
```

Comment:
Agree with @lazedo that these should go in the kapi module. You cannot call functions in one KAZOO app from another as that would require ecallmgr be installed alongside blackhole (in this case) which is not always desirable/allowed.